### DISCUSSION TITLE
我的 2018 盘点 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#211] [Announcements]
https://mazhuang.org/2019/01/20/my-2018/ 

 又到了回顾以明得失，展望以知进退的时候。

**Comment by sauchye** [comment]
壮哥好，打搅下，非常感谢你的blog模板，然后我clone后，发现gitalk无法使用，重新注册了repo以及key，提交更新后，博客也不更新😂，希望壮哥可以喵喵，感谢！https://sauchye.github.io/

**Comment by mzlogin** [comment]
@sauchye 提交后博客不更新是因为编译失败了，这种情况你的最近一次提交记录后面会有个 X，你点击会提示你具体的失败原因，然后你针对性的修改好就行了。

![image](https://user-images.githubusercontent.com/1646590/53294525-31faaa00-3824-11e9-818d-552b40d0f1bc.png)

gitalk 的配置我暂时看不出来有什么问题，等编译通过了再看看吧。

**Comment by sauchye** [comment]
感谢壮哥提醒，已搞定，链接哪里不能全部对齐= =

**Comment by ghost** [comment]
都是

**Comment by yizibi** [comment]
祝，在管理的路上越走越远，人总是要有危机意识的，我跟你一样，认识到技术路线的先天不足，耐不住性子一直研究底层，市场从来不缺高手，难的是，把自己练就成一个高手，祝，越来越好！

**Comment by fengerzh** [comment]
恭喜🎉🎉🎉祝贺🎊🎊🎊三个选择都是无比正确的：结婚、升官、旅游。技术人员如果不能经常出外走走会限制自己的视野；走经理之路是为人类做更大贡献的必由之路，前三十年学习，后三十年教书育人；结婚就不用多说了，早点有个孩子，你能从教育孩子身上学到该如何管理别人。

**Comment by mzlogin** [comment]
> @fengerzh
> 恭喜🎉🎉🎉祝贺🎊🎊🎊三个选择都是无比正确的：结婚、升官、旅游。技术人员如果不能经常出外走走会限制自己的视野；走经理之路是为人类做更大贡献的必由之路，前三十年学习，后三十年教书育人；结婚就不用多说了，早点有个孩子，你能从教育孩子身上学到该如何管理别人。

谢谢张老师鼓励，继续努力:smile:

**Comment by Xiaocai9527** [comment]
赞赞赞！

**Comment by Xiaocai9527** [comment]
小马哥牛皮（破音），嘿嘿嘿。

**Comment by Leejunxiang** [comment]
哇 您结婚了呀 祝福您。我两年前clone过您的模板，不知道您还有没有印象，今天想起来点进来，发现了您这样的喜事，虽然素不相识，但依然为您而感觉开心

**Comment by mzlogin** [comment]
> @Leejunxiang
> 哇 您结婚了呀 祝福您。我两年前clone过您的模板，不知道您还有没有印象，今天想起来点进来，发现了您这样的喜事，虽然素不相识，但依然为您而感觉开心

谢谢谢谢，对你 ID 有点印象的哈哈~

**Comment by huanrenwang** [comment]
clone了你一个模板，谢谢你的模板

**Comment by bemvp** [comment]
## 加油
感谢你的模板，我也用来写自己的总结，复盘
也写自己的年目标
转行的焦虑我也有，之前搜集了不少资料在我自己的规划中[我的整体思路和规划](https://mubu.com/doc/bXbKcSO7z)
分享出来希望有点启发
也希望相互交流
